test/libdnf5/repo/test_repo.cpp

```diff
@@ -138,3 +138,54 @@ void RepoTest::test_load_repo_nonexistent() {
     repos.filter_id(repoid);
     CPPUNIT_ASSERT_THROW(repo_sack->update_and_load_repos(repos), libdnf5::repo::RepoDownloadError);
 }
+
+void RepoTest::test_load_repo_twice() {
```

Comment:
Just last request to change from me. We can drop this test. Unfortunately, AFAIK we cannot easily test through API that the given repo is already loaded. The current test is checking some counters for loading metadata, but these are skipped anyway even when calling repo->load() multiple times. So better just delete this test and keep only the `test_update_and_load_enabled_repos_twice_fails` here.